deploy/nfpm/hoprd.initd

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
+
+description="HOPR node service"
+command="/usr/bin/hoprd"
+command_args="--configurationFilePath /etc/hoprd/hoprd.cfg.yaml"
```

Comment:
Should this not be an env variable as well?

Reply:
No, there are some hardcoding in the package like installation paths and configuration files path.
/etc/hoprd/hoprd.cfg.yaml it's a must, as the installation script will rely on that as well.